### PR TITLE
Add a part about Oh My ZSH themes libraries

### LINKED ISSUE #508: [docs]: Migration improvents
### Description

Most OMZ themes are relying on OMZL functions (eg. `prompt_info_functions.zsh` or `theme-and-appearance.zsh` for theme coloring...) to function. 
But, if those library snippets aren't loaded in `zi` it fails with a cryptic error (or no error at all and broken theme)

For example `3den` and many other send an error message with :

```bash
zsh: command not found: ruby_prompt_info
```

It would appear that it needs an executable that isn't present. But, after looking at the code, it was needed to load `OMZL::prompt_info_functions.zsh` to make that error disappear.

The most impactful is the `OMZL::theme-and-appearance.zsh` because, without it, the theme will have no color.

I suggest that a warning be added to the `Migration` page to tell users to load the appropriate libraries in order for their themes to work.

### Select the area that is associated with this issue.

ZI (this repository)

### Self-service

- [X] I'd be willing to address this documentation request myself.

### Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/z-shell/community/blob/main/docs/CONTRIBUTING_GUIDELINES.md).

### Are you familiar with Contributor Covenant Code of Conduct?

- [X] I have read the [Contributor Covenant Code of Conduct](https://github.com/z-shell/zi/blob/main/docs/CODE_OF_CONDUCT.md).

### Contact Details

stigpro@outlook.fr

### PR DESCRIPTION
**Pull request by signed-log**
OMZ themes need their support libraries to function.

Add a note about them

Fixes #508

### CODE REVIEW

**Review by ss-o — CHANGES_REQUESTED** [review]

File: docs/getting_started/03_migration.md [hunk 1]
```diff
@@ -178,14 +179,41 @@ If the `Git` library is not loaded or loaded in the wrong order, then it may app
...
 ```shell showLineNumbers
 zi snippet OMZL::git.zsh
 zi snippet OMZP::git
 zi cdclear -q
+zi snippet OMZL::theme-and-appearance.zsh
+zi snippet OMZL::prompt_info_functions.zsh
+# Other libraries that might be needed
 ```
 
+:::tip
```
[the remark this answers is not in the document]
Comment (signed-log) [reply docs/getting_started/03_migration.md:205]: I think that is better to leave that out entirely actually

File: docs/getting_started/03_migration.md [hunk 2]
```diff
@@ -178,11 +179,26 @@ If the `Git` library is not loaded or loaded in the wrong order, then it may app
 ........:1: command not found: git_prompt_short_sha
 ```
 
+If you encounter any issue with the theme, OMZ support libraries are to be loaded
+
+- If your theme isn't colored when it should, you will want to load `theme-and-appearance.zsh`
+
+- If you encounter an error message similar to :
```
Comment (ss-o) [inline docs/getting_started/03_migration.md:186]: You missed one :see_no_evil: 

About the rest is all fine by me. :100:

**Review by ss-o — CHANGES_REQUESTED** [review]

File: docs/getting_started/03_migration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ss-o) [reply docs/getting_started/03_migration.md]: It's up to you :+1:

I just propose my opinion and it's totally up to the community to shape it as they are the ones who will use it :octocat:

**Review by ss-o — APPROVED** [review]
Most important all the docs are up to date and follow rules to be compatible, the rest can be easy adjusted with various tools :octocat: :+1:

**Comment by ss-o** [comment]
Looks legit, thanks for taking the time :1st_place_medal:

**Pull request merged**